instance/parameters.go

```diff
@@ -17,15 +17,17 @@
 package instance
 
 import (
-	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient"
-	"net/http"
-	"fmt"
 	"errors"
+	"fmt"
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient"
 	"io/ioutil"
+	"net/http"
 )
 
-func Parameters(authClient httpclient.AuthenticatedClient, serviceInstanceAdminURL string, accessToken string) (string, error) {
-	bodyReader, statusCode, err := authClient.DoAuthenticatedGet(serviceInstanceAdminURL+"/parameters", accessToken)
+type ParametersOperation struct{}
```

Comment:
Let's not export these struct types from the package. The only type that needs to be exported is the `Operation` interface - the implementor structs can be kept package-private, I think.

Reply:
Yes, you're right